_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -38,62 +42,42 @@ const UpgradeNoticeContent = withModuleSettingsFormHelpers(
 		};
 
 		renderInnerContent() {
-			const blockEditorUrl = `${ this.props.adminUrl }post-new.php`;
+			/*eslint-disable react/no-danger*/
 			return (
 				<div className="jp-upgrade-notice__content">
-					<p>
-						{ __( 'The features you rely on, adapted for the new WordPress editor.' ) }
-						<br />
-						{ __( 'A new editor? Yes! {{a}}Learn more{{/a}}.', {
-							components: {
-								a: (
-									<ExternalLink
-										target="_blank"
-										rel="noopener noreferrer"
-										href={ 'https://wp.me/p1moTy-cee' }
-									/>
-								),
-							},
-						} ) }
-					</p>
-
-					<h2>{ __( 'Build your Jetpack site with blocks' ) }</h2>
-
-					<p>
-						{ __(
-							'Today, we are introducing the first wave of Jetpack-specific blocks built specifically ' +
-								'for the new editor experience: Simple Payment button, Form, Map, and Markdown.'
-						) }
-					</p>
-					<p>
-						<img
-							src={ imagePath + 'block-picker.png' }
-							width="250"
-							alt={ __( 'Jetpack is ready for the new WordPress editor' ) }
-						/>
-					</p>
+					<div dangerouslySetInnerHTML={ { __html: this.props.releasePostContent } } />
```

Comment:
I think it may be worth adding a comment here for folks who may stumble across this later. The post content is sanitized before it gets here.

Reply:
Good point! I added a comment in commit 4d71be6.